src/Schema/SQLServerSchemaManager.php

```diff
@@ -272,26 +296,6 @@ protected function _getPortableViewDefinition($view)
         return new View($view['name'], $view['definition']);
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function listTableIndexes($table)
-    {
-        $sql = $this->_platform->getListTableIndexesSQL($table, $this->_conn->getDatabase());
-
-        try {
-            $tableIndexes = $this->_conn->fetchAllAssociative($sql);
-        } catch (Exception $e) {
-            if (strpos($e->getMessage(), 'SQLSTATE [01000, 15472]') === 0) {
```

Comment:
This logic was added in 03be671e1df6338a7b528646fb51bea537dd44b7. Judging by the build, it's not necessary any longer or isn't properly tested.